pkg/apis/intevents/v1/pullsubscription_lifecycle.go

```diff
@@ -93,14 +95,21 @@ func (s *PullSubscriptionStatus) MarkDeployedUnknown(reason, messageFormat strin
 
 // PropagateDeploymentAvailability uses the availability of the provided Deployment to determine if
 // PullSubscriptionConditionDeployed should be marked as true or false.
-func (s *PullSubscriptionStatus) PropagateDeploymentAvailability(d *appsv1.Deployment) {
+// For authentication check purpose, this method will return true  if a false condition
```

Comment:
I would normally expect it to be the opposite, where `true` is everything is healthy.